ci_tests/asciidoc_validation/run_tests.sh

```diff
@@ -77,4 +77,22 @@ run_test "test_bad_cross_ref" \
   "S100 cross-references .*rules/S1000/bad.adoc" \
   "S1000 cross-references .*rules/S100/java/bad.adoc"
 
+run_test "test_diff_source" \
+  "ERROR: Diff sources are incorrectly used:" \
```

Comment:
"diff sources" is something new. I've heard the term "diff view" and "diff highlighting"
Also, I would inverse "incorrectly used", that sounds better subjectively, but I don't know if there is any English rule about it.
```suggestion
  "ERROR: Diff highlighting is used incorrectly:" \
```